File: Week2/day2/draw-square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was

number = int(input('Give me a number, please: '))
side = "%%%%%"
if number == 1:
    print(side)
elif number == 2:
    print(side)
    print(side)
elif number != 2 and number != 3:
    print(side)
    for x in range(1, number-1):
        print('%   %')
    print(side)
elif number == 3:
    print(side)
    print('%   %')
    print(side)

File: Week7/Day2/draw-square.js
'use strict';

var lineCount = 6;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

var space = '';
var mark = "%%";
for (var j = 1; j <= lineCount-2; j++)
{
    space += " ";
    mark += "%";
}
console.log(mark);
for (var i = 1; i <= lineCount-2; i++)
{
    console.log("%" + space + "%");
}
console.log(mark);